package.json

```diff
@@ -413,7 +415,8 @@
 		"@wordpress/viewport": "6.8.0",
 		"@wordpress/warning": "3.8.0",
 		"@wordpress/widgets": "4.8.0",
-		"@wordpress/wordcount": "4.8.0"
+		"@wordpress/wordcount": "4.8.0",
+		"typescript": "5.3.3"
```

Comment:
In order for React Scanner to pick our TS version up, we need to specifically pin the resolution version here.